server/wfnews-model/src/main/java/ca/bc/gov/nrs/wfnews/api/model/v1/SimplePublishedIncident.java

```diff
@@ -90,6 +90,9 @@ public interface SimplePublishedIncident {
 	public String getFireCentre();
```

Comment:
Now that the DB column name has changed, we should also change the attribute in the Java model.

This will mean updating the getter/setters and names in the model interface and resources, and should be updated in the factory, dto, etc.

Reply:
Thank you, done